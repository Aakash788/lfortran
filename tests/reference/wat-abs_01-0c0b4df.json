{
    "basename": "wat-abs_01-0c0b4df",
    "cmd": "lfortran --no-color --show-wat {infile}",
    "infile": "tests/../integration_tests/abs_01.f90",
    "infile_hash": "36697d1e0196bbf7a46059f539b539a1454386697ab54a43e3c32791",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "wat-abs_01-0c0b4df.stdout",
    "stdout_hash": "a258ccc996161022776921458aceb39e7f434e9e3643dd5fd0594a1a",
    "stderr": "wat-abs_01-0c0b4df.stderr",
    "stderr_hash": "7f7c146103a4cd452ef5811afe3466943854173e660065814ea214a3",
    "returncode": 0
}