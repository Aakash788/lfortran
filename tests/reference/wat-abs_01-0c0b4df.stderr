warning: Function with no body
 --> tests/../integration_tests/abs_01.f90:1:1
  |
1 | program abs_01
  | ^ char

warning: WASM: C Intrinsic Functions not yet spported
 --> tests/../integration_tests/abs_01.f90:1:1
  |
1 | program abs_01
  | ^ char

warning: Function with no body
 --> tests/../integration_tests/abs_01.f90:1:1
  |
1 | program abs_01
  | ^ move_alloc

warning: WASM: C Intrinsic Functions not yet spported
 --> tests/../integration_tests/abs_01.f90:1:1
  |
1 | program abs_01
  | ^ move_alloc

warning: WASM: C Intrinsic Functions not yet spported
 --> tests/../integration_tests/abs_01.f90:1:1
  |
1 | program abs_01
  | ^ cpu_time

warning: WASM: C Intrinsic Functions not yet spported
 --> tests/../integration_tests/abs_01.f90:1:1
  |
1 | program abs_01
  | ^ dp_rand_num

warning: WASM: C Intrinsic Functions not yet spported
 --> tests/../integration_tests/abs_01.f90:1:1
  |
1 | program abs_01
  | ^ i32sys_clock

warning: WASM: C Intrinsic Functions not yet spported
 --> tests/../integration_tests/abs_01.f90:1:1
  |
1 | program abs_01
  | ^ i64sys_clock

warning: WASM: C Intrinsic Functions not yet spported
 --> tests/../integration_tests/abs_01.f90:1:1
  |
1 | program abs_01
  | ^ sabs

warning: WASM: C Intrinsic Functions not yet spported
 --> tests/../integration_tests/abs_01.f90:1:1
  |
1 | program abs_01
  | ^ sp_rand_num
